.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Team Lead
+* @rosle
+
+# Team Members
+* @andyduong1920 @bterone @byhbt @carryall @hanam1ni @hoangmirs @junan @Lahphim @longnd @malparty @tyrro
+
+# Engineering Leads
+CODEOWNERS @nimblehq/engineering-leads
```

Comment:
Should we add an ending empty line? :)

Reply:
We sure can but it is definitely not needed 😁  I will add it since it is our convention for all files. Added in [14be9c3](https://github.com/nimblehq/rails-templates/pull/328/commits/14be9c3a54df7c8055a4fcca3166cc5ccd879ccd) but only visible in the IDE-view of the PR:

![image](https://user-images.githubusercontent.com/696529/160553759-e4ea64b8-fc87-4e5d-8697-036a0dfdfa71.png)